multi-int-store/src/main/java/com/connexta/multiintstore/config/ApplicationConfiguration.java

```diff
@@ -32,4 +42,27 @@ public CommonsRequestLoggingFilter requestLoggingFilter() {
     filter.setMaxPayloadLength(5120);
     return filter;
   }
+
+  @Bean
+  public SearchService searchService(
+      IndexedMetadataRepository indexedMetadataRepository,
+      @Value("${endpointUrl.retrieve}") String retrieveEndpoint) {
+    return new SearchServiceImpl(indexedMetadataRepository, retrieveEndpoint);
+  }
+
+  @Bean
+  public IndexedMetadataDao indexedMetadataDao(IndexedMetadataRepository repository) {
+    return new IndexedMetadataDao(repository);
```

Comment:
✏️ Should we have interfaces and impls for `IndexedMetadataDao`, `ProductStorageManager`, and `MetadataStorageManager`? We can probably do this in another PR.

Reply:
I would say we should wait until we need an interface. I'm not sure what practical value we would be adding by making an interface for these.